libs/labelbox/pyproject.toml

```diff
@@ -73,7 +73,7 @@ dev-dependencies = [
 ]
 
 [tool.rye.scripts]
-unit = "pytest tests/unit"
+unit = "pytest tests/unit libs/labelbox/tests/unit"
```

Comment:
I don't think these are needed, the unit context only exists when the current directory is in the libs/labelbox folder

```
achang@Adrians-MacBook-Pro-2 ~/D/labelbox-python (achang/plt-391-lib-upgrade)> rye run
coverage
coverage3
dmypy
docs (sphinx-build ./docs ./dist)
docutils
jsonschema
jupyter
jupyter-dejavu
jupyter-execute
jupyter-kernel
jupyter-kernelspec
jupyter-migrate
jupyter-nbconvert
jupyter-run
jupyter-troubleshoot
jupyter-trust
mypy
mypyc
normalizer
py.test
pybabel
pygmentize
pyrsa-decrypt
pyrsa-encrypt
pyrsa-keygen
pyrsa-priv2pub
pyrsa-sign
pyrsa-verify
pytest
python
python3
python3.8
rst2html.py
rst2html4.py
rst2html5.py
rst2latex.py
rst2man.py
rst2odt.py
rst2odt_prepstyles.py
rst2pseudoxml.py
rst2s5.py
rst2xetex.py
rst2xml.py
rstpep2html.py
sphinx-apidoc
sphinx-autogen
sphinx-build
sphinx-quickstart
stubgen
stubtest
toml
tqdm
yapf
```
```
achang@Adrians-MacBook-Pro-2 ~/D/l/l/labelbox (achang/plt-391-lib-upgrade)> rye run
coverage
coverage3
data (pytest tests/data)
dmypy
docutils
integration (pytest tests/integration)
jsonschema
jupyter
jupyter-dejavu
jupyter-execute
jupyter-kernel
jupyter-kernelspec
jupyter-migrate
jupyter-nbconvert
jupyter-run
jupyter-troubleshoot
jupyter-trust
lint (chain: [yapf-lint], [mypy-lint])
mypy
mypy-lint (mypy src --pretty --show-error-codes --non-interactive --install-types)
mypyc
normalizer
py.test
pybabel
pygmentize
pyrsa-decrypt
pyrsa-encrypt
pyrsa-keygen
pyrsa-priv2pub
pyrsa-sign
pyrsa-verify
pytest
python
python3
python3.8
rst2html.py
rst2html4.py
rst2html5.py
rst2latex.py
rst2man.py
rst2odt.py
rst2odt_prepstyles.py
rst2pseudoxml.py
rst2s5.py
rst2xetex.py
rst2xml.py
rstpep2html.py
sphinx-apidoc
sphinx-autogen
sphinx-build
sphinx-quickstart
stubgen
stubtest
test (chain: [lint], [unit], [integration])
toml
tqdm
unit (pytest tests/unit)
yapf
yapf-diff
yapf-lint (yapf tests src -i --verbose --recursive --parallel --style google)
```